content-scripts/components/HeaderLinks.tsx

```diff
@@ -0,0 +1,26 @@
+import jsx from "texsaur";
+
+interface Props {
+  links: Record<string, Record<string, string>>;
+}
+
+const HeaderLinks = ({ links }: Props) => {
+  return (
+    <nav id="se-header-links">
+      {Object.entries(links).map(([key, value]) => (
+        <div className="se-header-link" key={key}>
+          <button>{key}</button>
```

Comment:
Regarding the buttons we could look into changing up their styling a bit in order to better fit into the direction UP's websites are going, such as [FCUP's](https://www.up.pt/fcup/en/) . Maybe aligning it to one side or playing with `text-transform: uppercase;`.

Reply:
Yeah, I think having a better beginning design can be interesting and might do for an interesting first issue. We'll leave this issue in particular do another PR.